src/coffee/sync/inventory-sync.coffee

```diff
@@ -20,7 +21,7 @@ InventoryUtils = require './utils/inventory'
 #     # do nothing
 class InventorySync extends BaseSync
 
-  @actionGroups = ['quantity', 'expectedDelivery']
+  @actionGroups = ['quantity', 'expectedDelivery', 'custom']
```

Comment:
I remembered you briefly explained what this does to me. Can you add it as a comment as I cannot remember again 😞